Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,46 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            while (InteractiosnManager.ContinueRunningProgram)
+            {
+                InteractiosnManager.PerformInteractions();
+            }
+        }
+    }
+
+    public static class InteractiosnManager
+    {
+        public static bool ContinueRunningProgram = true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)